plugins/essif-lab_contactform7/src/js/script.js

```diff
@@ -0,0 +1,25 @@
+(function( $ ) {
+	$( window ).load(function() {
+		$("#essif-lab").click(function (e) {
```

Comment:
Deze selector is niet hook & target specifiek. Het moet sws iets van `.cf7 #essif-lab` zijn (dus hook specifiek). Maar uiteraard kan het ook zijn dat een gebruiker twee formulieren op 1 pagina toont die beide een validation policy hebben en dus 2 buttons getoond worden die allebei als het goed is hetzelfde doen.